# This message is a container for a Turtlebot's MMA-generated name, and its Occupancy Grid topic.
string name
nav_msgs/OccupancyGrid map